website/docs/r/compute_attached_disk.html.markdown

```diff
@@ -0,0 +1,109 @@
+---
+layout: "google"
+page_title: "Google: google_compute_attached_disk"
+sidebar_current: "docs-google-compute-attached-disk"
+description: |-
+  Resource that allows attaching existing persistent disks to compute instances.
+---
+
+# google\_compute\_attached\_disk
+
+Persistent disks can be attached to a compute instance using [the `attach_disk`
+section within the compute instance configuration](https://www.terraform.io/docs/providers/google/r/compute_instance.html#attached_disk).
+However there may be situations where managing the attached disks via the compute
+instance config isn't preferable or possible. For example: attaching dynamic
```

Comment:
I'm slightly anti-example just because it becomes obsolete in tf 0.12. If you'd like it to remain though, then can you rephrase into a complete sentence? (I'd just do "preferable or possible, such as attaching")